src/ui-lib/sass/05-components/02-molecules/00-navigation/_00-nav-menu.scss

```diff
@@ -38,13 +38,15 @@
     }
   }
 
+  li:last-child {
```

Comment:
Using a negative right margin on the `ul` will be a more robust solution.

As it stands, if the nav links wrap over multiple lines, then only the last link in the last row will have its right margin removed. The right-most links in preceding rows will still have a right margin.

If you use a negative margin on the `ul`, then it will work for _all_ rows of links.